### PR TITLE
[TAS-620] 👔 Add a WNFT CTA banner on /civic

### PR DESCRIPTION
**Pull request by AuroraHuang22**
截圖 2024-01-11 晚上10 57 18
截圖 2024-01-11 晚上10 57 25
截圖 2024-01-11 晚上11 05 07

### CODE REVIEW

File: src/components/CivicLikerV3/Page.vue [hunk 1]
```diff
@@ -1,50 +1,109 @@
...
+            :text="$t('civic_dashboard_v3_banner_book_nft')"
+          />
+          <Label
+            class="text-like-cyan font-200 mb-[24px]"
+            :text="$t('civic_dashboard_v3_banner_book_nft_description')"
+          />
+          <ButtonV2
+            preset="secondary"
+            :to="localeLocation({ name: 'store' })"
+            :text="$t('civic_dashboard_v3_banner_button')"
```
Comment (williamchong) [inline src/components/CivicLikerV3/Page.vue:57]: Can we also have a click event here for analytics?

Comment (AuroraHuang22) [reply src/components/CivicLikerV3/Page.vue:57]: [📈 Add redirect event for /civic](https://github.com/likecoin/liker-land/pull/1493/commits/be7e064409127dd45963b31c7c9a94081af5cc5c)

**Comment by AuroraHuang22** [comment]
> @AuroraHuang22 How did you extract the graphics? The wave lines are blocking the text in the English version. Additionally, there should be a grain effect for the background.
> 
> In my opinion, the banner with a whiter background would stand out more. Currently, the background color of the banner and the page background are almost the same.

[💄 Update banner bg style](https://github.com/likecoin/liker-land/pull/1493/commits/32174205ea5756e5c413f7a985bd3c9707d7da28)
截圖 2024-01-12 上午10 51 19

截圖 2024-01-12 上午10 51 31

**Comment by nwingt** [comment]
@AuroraHuang22 The wave lines seem not vertical-align with the key art

**Comment by AuroraHuang22** [comment]
Is it better this way? @nwingt 
[💄 Update banner bg style](https://github.com/likecoin/liker-land/pull/1493/commits/f60b4d84792de22a91da590aa3984d88104ba65e)

截圖 2024-01-12 下午2 10 46
截圖 2024-01-12 下午2 10 40

**Review by nwingt — APPROVED** [review]

**Pull request merged**